src/pages/Edit.js

```diff
@@ -1,65 +1,37 @@
-import React,{useState,useEffect} from "react";
-import {useParams,useHistory} from 'react-router-dom'
-import {Form, Button,Col} from 'react-bootstrap'
+import React, { useState, useEffect } from "react";
+import { Badge, Button, Row, Col } from "react-bootstrap";
+import { faMapMarker } from "@fortawesome/free-solid-svg-icons";
+import { faCalendar } from "@fortawesome/free-solid-svg-icons";
+import { faDollarSign } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useParams, Link } from "react-router-dom";
 
-export default function Edit() {
-    const {id} = useParams()
-    const history = useHistory()
-    let [job,setJob] = useState(null)
+import moment from "moment";
 
-    const getData = async() =>{
-        // let url = `http://localhost:3001/jobs/${id}`
-        let url=`https://my-json-server.typicode.com/legobitna/Itviec/jobs/${id}`
-     
-        let data = await fetch(url)
-        let result = await data.json();
-        console.log("Rr",result)
-        setJob(result)
-    }
-    useEffect(()=>{
-        getData();
-    },[])
+export default function Details() {
+  const { id } = useParams();
+  let [job, setJob] = useState(null);
 
-    const editJob=async (e)=>{
-        e.preventDefault();
-        const config = {
-            method: "PUT",
-            body: JSON.stringify(job),
-            headers: {
-              "Content-Type": "application/json"
-            }
-          };
-          try {
-            const response = await fetch(
-            //   `http://localhost:3001/jobs/${job.id}`,
-            ` https://my-json-server.typicode.com/legobitna/Itviec/jobs/${id}`,
-              config
-            );
-            console.log("Rr",response)
-            history.push("/")
-          } catch (error) {
-            console.log("Oops");
-          }
-    }
-    const tagChange = (e)=>{
-        let string = e.target.value;
-        let splitString = string.split(',')
-        setJob({...job,tags:splitString})
-    }
-    const benefitChange = (e,idx)=>{
-        let array = job.benefit;
-        array[idx] =  e.target.value;
-        setJob({...job,benefit:array})
-    }
-  
-    if(job===null){
-        return<div>No data yet</div>
-    }
+  const getData = async () => {
+    let url = `http://localhost:3001/jobs/${id}`
+    //let url = `https://my-json-server.typicode.com/legobitna/Itviec/jobs/${id}`;
 
+    let data = await fetch(url);
+    let result = await data.json();
+
+    setJob(result);
+  };
+  useEffect(() => {
+    getData();
+  }, []);
+
+  if (job === null) {
+    return <div>No data yet</div>;
+  }
   return (
     <div className="App">
-      <div className="searchHeader-login">
-        <div className="pageMenu">
+      <div className="search-header-login">
+        <div className="page-menu">
           <img
```

Comment:
this logo doesn't shows up with the wrong ratio (it looks squished) on my computer

Reply:
we will not gonna use edit so i will remove the Edit.js

Reply:
okay, please remove